components/kubeconfig-service/go.mod

```diff
@@ -71,31 +71,33 @@ require (
 )
 
 replace (
-	k8s.io/api => k8s.io/api v0.23.5
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.23.5
+	github.com/emicklei/go-restful => github.com/emicklei/go-restful/v3 v3.7.5-0.20220308211933-7c971ca4d0fd
+	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
```

Comment:
I think we noticed that there are two CVEs, and one of them is only fixed in a March 15 version, which is referenced by the github.com/kubernetes/kubernetes master branch

Reply:
yes you're right. emicklei/go-restful package i've fixed and using the later version v3.8.0 which the v.1.25.0-alpha version of kubernetes use. We have seen that make build-local has successfully ran.